src/fullscreen/fullscreen-controller.js

```diff
@@ -0,0 +1,255 @@
+//@flow
+import EventManager from '../event/event-manager';
+import Player from '../player';
+import FakeEvent from '../event/fake-event';
+import * as Utils from '../utils/util';
+
+/**
+ * The IOS fullscreen class name.
+ * @type {string}
+ * @const
+ */
+const IN_BROWSER_FULLSCREEN_FOR_IOS: string = 'playkit-in-browser-fullscreen-mode';
+
+/**
+ * @class FullscreenController
+ * @param {Player} player - The player.
+ */
+class FullscreenController {
+  _player: Player;
+  _inBrowserFullscreen: boolean;
+  _isInBrowserFullscreen: boolean;
+
+  /**
+   * after component mounted, set up event listeners to window fullscreen state change
+   * @param {Player} player - element to enter fullscreen
+   * @memberof FullScreenController
+   * @returns {void}
+   */
+  constructor(player: Player): void {
+    this._player = player;
+    this._inBrowserFullscreen = this._player.config.playback.inBrowserFullscreen;
```

Comment:
I would rename to `_inBrowserFullscreenConfig` because it confuses with `_isInBrowserFullscreen`
(Actually you can delete it since you use it only 2 times)

Reply:
It much nicer to work with this._inBrowserFullscreen and not  this._player.config.playback.inBrowserFullscreen